src/components/contextual/pages/pool/PoolChart.vue

```diff
@@ -117,33 +120,66 @@ const timestamps = computed(() =>
 
 function getTVLData(periodSnapshots: PoolSnapshot[]) {
   const tvlValues: (readonly (string | number)[])[] = [];
-  periodSnapshots.forEach((snapshot, idx) => {
-    const timestamp = timestamps.value[idx];
-    // get today's TVL value from pool.totalLiquidity due to differences in prices during the day
-    if (idx === 0) {
-      tvlValues.push(
-        Object.freeze([timestamp, Number(props.totalLiquidity || 0)])
-      );
-      return;
-    }
 
-    const prices = props.historicalPrices[snapshot.timestamp];
+  // temporary statement until we start get prices from coingecko for
+  if (props.poolAddress === BALANCER_BOOSTED_POOL_ADDRESS) {
```

Comment:
It would be better do to `if (isStablePhantom(props.pool.poolType)` rather than hard coding the boosted pool address, just in case we never get data in coingecko and we will probably see this pop up for future boosted pools.